extension/src/util/object.ts

```diff
@@ -11,3 +11,6 @@ export const createTypedAccumulator = <T extends string>(
   }
   return acc
 }
+
+export const isObject = (value: unknown): value is Record<string, unknown> =>
+  !!(value && !Array.isArray(value) && typeof value === 'object')
```

Comment:
[F] This looks like a duplicate of `isValueTree`. 

[Q] Not massively important but can we reuse this there?

i.e. 
```
export const isValueTree = (
  value: Value | ValueTree
): value is NonNullable<ValueTree> =>
  isObject(value)
```

Reply:
Reused `isValueTree`